packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -251,19 +250,23 @@ export class SqlPluginImpl {
     sqlClient: SqlClient,
     managementClient: ManagementClient
   ): Promise<void> {
+    let retryAddUser = 0;
```

Comment:
I may suggest better variable naming here, e.g. retryCount or something else

Reply:
It is fixed. Thanks.